### ISSUE TITLE
$this->registerHook('entry_before_insert', [$this, 'handleBeforeInsert']); Duplicate entries

### ISSUE THREAD

**Issue by iferencz** [FreshRSS/Extensions#233]
```
$this->registerHook('entry_before_insert', [$this, 'handleBeforeInsert']);
```
I use this hook to handle new entries, but sometimes the `handleBeforeInsert` function receives duplicate entries that already exist in the database. Has anyone else encountered this problem? I have `return $entry;` at the end of the function.Thanks.

**Comment by Alkarex** [comment]
Hello,
This is by design. Updated entries might have to be processed by extensions just like new entries.
On the `edge` branch (soon released in 1.24.0), you can test `$entry->isUpdated()`, otherwise see https://github.com/FreshRSS/FreshRSS/discussions/5602

**Comment by iferencz** [comment]
> Hello, This is by design. Updated entries might have to be processed by extensions just like new entries. On the `edge` branch (soon released in 1.24.0), you can test `$entry->isUpdated()`, otherwise see [FreshRSS/FreshRSS#5602](https://github.com/FreshRSS/FreshRSS/discussions/5602)

Thanks!
